mlir/test/fusion/linalg-generic-reuse-buffers.mlir

```diff
@@ -0,0 +1,60 @@
+// RUN: rocmlir-driver --rock-linalg-align %s |./external/llvm-project/llvm/bin/FileCheck %s
+#map1 = affine_map<(d0, d1, d2) -> (d0, d1, d2)>
+#map2 = affine_map<(d0, d1, d2, d3, d4, d5, d6, d7) -> (d1, d0 * 16 + d5 + d7, (d2 * 16 + d4) * 4 + d6)>
+#map3 = affine_map<(d0, d1, d2, d3, d4, d5) -> (d0, d1, d2, d3, d4 floordiv 16, d4 mod 16, d5, 0)>
+
+// CHECK: [[MAP:.*]] = #rock.transform_map<#map by [<PassThrough ["g_block"] at [1] -> ["g"] at [0]>, <Unmerge{4, 16, 1} ["k_loop", "k_thread", "k_iter"] at [0, 5, 7] -> ["k"] at [1]>, <Unmerge{1, 16, 4} ["m_block", "m_thread", "m_iter"] at [2, 4, 6] -> ["m"] at [2]>, <AddDim{1} ["n_block"] at [3] -> [] at []>] bounds = [4, 64, 1, 1, 16, 16, 4, 1] -> [64, 64, 64]>
```

Comment:
You'll probably want to `#{{.*}}` instead of `#map` or `#map1` here. Also `CHECK-DAG` on these